bindings/python/packaging/BUILD.bazel

```diff
@@ -23,6 +23,7 @@ py_binary(
     tags = [
         # Do not build with ... expansion
         "manual",
+        "no-rbe",
```

Comment:
Can you add a comment here explaining why this isn't compatible (or we just don't want to run it) with RBE?

Alternately, why not use `nokokoro`? Do we have Kokoro running both with and without RBE?

Reply:
Yes, will do.

Yes we currently do but that wasn't the reason. I was thinking of putting `no-rbe` because it's more descriptive of the problem and people use RBE outside of kokoro. I changed it to `nokokoro`.